readthedocsext/theme/templates/profiles/partials/github_oauth_disconnect_list.html

```diff
@@ -23,7 +23,7 @@
         action="{% url 'socialaccount_connections' %}">
     {% csrf_token %}
     <input type="hidden" name="account" value="{{ object.id }}" />
-    <button class="ui button" type="submit">
+    <button class="ui button" type="submit" disabled>
```

Comment:
I don't think there is difference on styles, but I'd still do this in case:

```suggestion
    <button class="ui disabled button" type="submit" disabled>
```